Main.h

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Give meaningful names to the split files. This header has color definitions, so could be called colors.h?
Find meaningful names for the other filenames as well.

Reply:
Thanks Sudeep!! I have made changes in the same.